src/main/java/org/takes/facets/auth/PsBasic.java

```diff
@@ -195,7 +204,7 @@ public Fake(final boolean cond) {
      * Default entry.
      *
      * @author Georgy Vlasov (wlasowegor@gmail.com)
-     * @version $Id$
+     * @version $Id: 5913e70e4877ecdd3f0cb85821dd8593f48a166a $
```

Comment:
@dalifreire remove the unneeded string at the end, this field should be `@version $Id$` only